runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/PTransformTranslation.java

```diff
@@ -120,13 +124,26 @@ private PTransformTranslation() {}
     return transformBuilder.build();
   }
 
+  /**
+   * Translates a non-composite {@link AppliedPTransform} into a runner API proto.
```

Comment:
"An AppliedPTransform with no components"?

Alternatively this could be "Translates an AppliedPTransform into a runner API proto with no component transforms.", and disclaims that this should not be used when translating a Pipeline